hack/generator/pkg/jsonast/jsonast.go

```diff
@@ -8,11 +8,12 @@ package jsonast
 import (
 	"context"
 	"fmt"
-	"github.com/Azure/k8s-infra/hack/generator/pkg/config"
 	"net/url"
 	"regexp"
 	"strings"
 
+	"github.com/Azure/k8s-infra/hack/generator/pkg/config"
```

Comment:
super minor: put this down by other other hack/generator/pkg imports?

Reply:
I'm surprised go fmt didn't do this. I don't manage the imports myself!